src/test/EvenNumbersGeneratorTests.java

```diff
@@ -0,0 +1,54 @@
+package test;
+
+import com.company.EvenNumbersGenerator;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import static org.assertj.core.api.Assertions.*;
+
+public class EvenNumbersGeneratorTests {
+    @Test
+    public void Min_One_Max_Tree_Results_In_Two()
+    {
+        EvenNumbersGenerator sut = new EvenNumbersGenerator();
+        List<Integer> expectedResult = Arrays.asList(2);
```

Comment:
you can use `expectedResult` -> `expected` and `actualResult` -> `actual`.

it has the same meaning but makes code more compact

Reply:
Done